### ISSUE TITLE
timeseries_weather_forecasting.py improvement/clarification

### ISSUE THREAD

**Issue by ramarro123** [keras-team/keras-io#1083]
Hello,

in the example here
https://keras.io/examples/timeseries/timeseries_weather_forecasting/
and his git file
https://github.com/keras-team/keras-io/blob/master/examples/timeseries/timeseries_weather_forecasting.py

it will be great if it will be possible to use standard data on show_plot not the normalized data, in this way the actual parameter (of temperature or whatever is used) can be displayed.

also, a comment where it's explained which index and where it's used for prediction feature will be good, in case we want to check what happen if we want to get Relative Humidity instead of Temperature (from my understanding should be enough to change here
y_val = features.iloc[label_start:][[1]] passing 1 as a variable but i am not sure)

warm regards

**Comment by dhantule** [comment]
Hi @ramarro123, thanks for reporting this. 

To plot the actual data, we need to reverse the normalization process, you do that by modifying `show_plot`

```
if scaler:
        plot_data[2] = scaler.inverse_transform(plot_data[2].reshape(-1, 1))  # Inverse transform model prediction
    
    # Plot the model prediction after inverse transform
    plt.plot(future, plot_data[2], marker[2], markersize=10, label="Model Prediction (Actual Scale)")

plt.legend()
plt.xlim([time_steps[0], (future + 5) * 2])
plt.xlabel("Time-Step")
plt.show()
```
You can update the target feature in the training set` y_train` and validation set `y_val ` with the respective index to predict for that target feature.
